src/Microsoft.Data.Analysis/GroupBy.cs

```diff
@@ -464,5 +466,12 @@ public override DataFrame Mean(params string[] columnNames)
             return ret;
         }
 
+        public IEnumerable<IGrouping<TKey, DataFrameRow>> Groupings
+        {
+            get 
+            {
+                return _keyToRowIndicesMap.Select(kvp => new Grouping<TKey>(kvp.Key, kvp.Value.Select(index => _dataFrame.Rows[index]).ToArray()));
```

Comment:
The `ToArray` here will make the computation here greedy right? Would it be better to keep this lazy (since the return type here is `IEnumerable<IGrouping<TKey, DataFrameRow>>`)?  I think just removing the `ToArray` will work since DataFrameRow implements `IEnumerable` (I may be slightly wrong here. I didn't try to compile this)

Reply:
fixed